cmd/operator-sdk/generate/csv.go

```diff
@@ -53,35 +55,58 @@ for the operator. This file is used to publish the operator to the OLM Catalog.
 
 A CSV semantic version is supplied via the --csv-version flag. If your operator
 has already generated a CSV manifest you want to use as a base, supply its
-version to --from-version. Otherwise the SDK will scaffold a new CSV manifest.
+version to --from-version. Otherwise the SDK will scaffold a new CSV manifest.`,
+		Example: `
+TODO
+	`,
 
-Configure CSV generation by writing a config file 'deploy/olm-catalog/csv-config.yaml`,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			// The CSV generator assumes that the deploy and pkg directories are
-			// present at runtime, so this command must be run in a project's root.
-			projutil.MustInProjectRoot()
```

Comment:
HI @hasbro17, 

If we execute the command from another dir which is not the root of the project it shows that worked but actually not. I know that the goal is to allow it to work with  no-sdk project as well. However, IMO we should check and add an WARN. 

```
        if err := CheckProjectRoot(); err != nil {
		// log warn such as `WARN[0000] Ensure that you are running this command from the project root dir. `
	}
```